Gemfile

```diff
@@ -28,7 +28,7 @@ gem 'jbuilder', '~> 2.5'
 # Use Redis adapter to run Action Cable in production
 gem 'redis', '~> 4.0'
 gem 'sidekiq', '~> 6.0.2'
-gem 'pundit'
+gem 'pundit', '~> 2.1.0'
```

Comment:
This was in another PR, doesn't it?

Reply:
Yes. I progressed in other tasks and after the review I was merging